src/Altinn.Platform/Altinn.Platform.Storage/Storage.Interface/Models/InstanceEvent.cs

```diff
@@ -0,0 +1,73 @@
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Altinn.Platform.Storage.Models
+{
+    /// <summary>
+    /// Model for the instance event
+    /// </summary>
+    public class InstanceEvent
+    {
+        /// <summary>
+        /// Gets or sets identifier used to identify unique instance events. 
+        /// </summary>
+        [JsonProperty(PropertyName = "id")]
+        public Guid? Id { get; set; }
+
+        /// <summary>
+        /// Gets or sets identifier used to identify unique instance
+        /// </summary>
+        [JsonProperty(PropertyName = "instanceId")]
+        public string InstanceId { get; set; }
+
+        /// <summary>
+        /// Gets or sets identifier used to identify when Authentication Event was created
+        /// </summary>
+        [JsonProperty(PropertyName = "createdDateTime")]
+        public DateTime? CreatedDateTime { get; set; }
```

Comment:
Why is this optional? timestamp. Who sets this client or storage?

Reply:
It is set in storage. Made nullable to avoid default date to be assigned.